pkg/controller/clusteringress/clusteringress_controller_test.go

```diff
@@ -95,6 +96,6 @@ func TestClusterIngressController(t *testing.T) {
 	}
 
 	assert.Equal(t, "5s", routes.ObjectMeta.Annotations[resources.TimeoutAnnotation])
-	assert.NotEqual(t, networkingv1alpha1.DefaultTimeout, routes.ObjectMeta.Annotations[resources.TimeoutAnnotation])
+	assert.NotEqual(t, 10*time.Minute, routes.ObjectMeta.Annotations[resources.TimeoutAnnotation])
```

Comment:
Upstream got rid of a constant here? We just have to hardcode the default upstream timeout?

Reply:
Isn't that only used in testing code anyway? We have a hardcoded fallback in the production-code anyway.